src/swarm/neo/client/requests/Stats.d

```diff
@@ -44,9 +100,10 @@ public interface IRequestStats
 
         ***********************************************************************/
 
+        deprecated("Use histogram.mean_time_micros instead")
         public double mean_handled_time_micros ( )
         {
-            return cast(double)this.total_time_micros / cast(double)this.count;
+            return this.histogram.mean_time_micros();
```

Comment:
You deprecate only this method, but not `count` or `total_time_micros`, for example? Also, deprecation note.

Reply:
Ah yes, of course. Updated.